frontend/www/js/omegaup/components/arena/ContestListv2.test.ts

```diff
@@ -235,6 +235,24 @@ describe('ContestListv2.vue', () => {
     expect(pastContestTab.text()).toContain('Past Contest 1');
   });
 
+  it('Should handle paginator', async () => {
+    const wrapper = mount(arena_ContestList, {
+      propsData: {
+        contests,
+        countContests: {
+          current: 24,
+          future: 0,
+          past: 56,
+        },
+        tab: ContestTab.Future,
+      },
+    });
+
+    const paginator = wrapper.findComponent({ ref: 'paginator' });
+
+    expect(paginator.text()).toBe('ddfd');
+  });
```

Comment:
Here you can use a [parameterised test](https://www.npmjs.com/package/jest-each) to cover all the functionallity:
```suggestion
  const periodMapping = [
    {
      tab: ContestTab.Current,
      expectedValue: '‹123›',
    },
    {
      tab: ContestTab.Future,
      expectedValue: '‹1›',
    },
    {
      tab: ContestTab.Past,
      expectedValue: '‹123456›',
    },
  ];

  each(periodMapping).it(
    'Should handle paginator when "%s" field is selected', 
    async ({tab, expectedValue}) => {
    const wrapper = mount(arena_ContestList, {
      propsData: {
        contests,
        countContests: {
          current: 24,
          future: 0,
          past: 56,
        },
        tab,
      },
    });

    const paginator = wrapper.findComponent({ ref: 'paginator' });

    expect(paginator.text()).toBe(expectedValue);
  });

```